templates/helpers.rb

```diff
@@ -314,6 +316,47 @@ def in_context name
     @convert_context.pop
     result
   end
+
+  STEM_EQNUMS_AMS = 'ams'
+  STEM_EQNUMS_NONE = 'none'
+  STEM_EQNUMS_VALID_VALUES = [
+    STEM_EQNUMS_NONE,
+    STEM_EQNUMS_AMS,
+    'all'
+  ]
+
+  # Generate the Mathjax markup to process STEM expressions
+  # @param cdn_base [String]
+  # @return [String]
+  def generate_stem(cdn_base)
+    if attr?(:stem)
+      eqnums_val = attr('eqnums', STEM_EQNUMS_NONE)
+      eqnums_val = STEM_EQNUMS_AMS if eqnums_val == ''
+      unless STEM_EQNUMS_VALID_VALUES.include?(eqnums_val)
+        raise ::ArgumentError, %(Invalid eqnums value [#{eqnums_val}], valid values are #{STEM_EQNUMS_VALID_VALUES.join(', ')})
```

Comment:
An Asciidoctor converter should not raise an error (unless something is truly, truly sideways). Rather, it should log a warning or error. This allows the outcome to be controlled by the user. If the user chooses to set the log failure level, then Asciidoctor will terminate with a non-zero exit code at the end.

In this case, it would be better to coerce the value to an acceptable value. We don't want to stop the conversion over such a petty problem. And we may not even need to warn.

Reply:
OK, I've changed the code to use the `ams` value and put no warning